### DISCUSSION TITLE
v0.9

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#661] [New Releases]
## What's Changed
* Add overtype toggling by @dail8859 in https://github.com/dail8859/NotepadNext/pull/589
* Search Box Positioning by @pydavem in https://github.com/dail8859/NotepadNext/pull/591
* Copy Search results #224 by @pydavem in https://github.com/dail8859/NotepadNext/pull/596
* Fix crashes with Qt 6.7.2 by @sandman7920 in https://github.com/dail8859/NotepadNext/pull/623
* Fix macro saving
* Adjust JSON error colors
* Add additional word chars by @dail8859 in https://github.com/dail8859/NotepadNext/pull/625
* Fix keywords for C by @dail8859 in https://github.com/dail8859/NotepadNext/pull/632
* Add backward direction to Find dialog by @accessv in https://github.com/dail8859/NotepadNext/pull/635
* Add loading of custom application styling by @dail8859 in https://github.com/dail8859/NotepadNext/pull/637
* Add option to sort file list by name by @dail8859 in https://github.com/dail8859/NotepadNext/pull/638
* Attempt to fix Win10 crash by @dail8859 in https://github.com/dail8859/NotepadNext/pull/639
* Update NotepadNext_sv_SE.ts by @eson57 in https://github.com/dail8859/NotepadNext/pull/606
* Updated Ukrainian translation by @YALdysse in https://github.com/dail8859/NotepadNext/pull/610
* Update Portuguese (Portugal) translation by @hugok79 in https://github.com/dail8859/NotepadNext/pull/579
* Update Traditional Chinese locale by @PeterDaveHello in https://github.com/dail8859/NotepadNext/pull/646
* Update Scintilla to v5.5.0 by @dail8859 in https://github.com/dail8859/NotepadNext/pull/587
* Update Lexilla to v5.3.2 by @dail8859 in https://github.com/dail8859/NotepadNext/pull/588

## New Contributors
* @pydavem made their first contribution in https://github.com/dail8859/NotepadNext/pull/591
* @sandman7920 made their first contribution in https://github.com/dail8859/NotepadNext/pull/623
* @accessv made their first contribution in https://github.com/dail8859/NotepadNext/pull/635

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.8...v0.9

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.9'>v0.9</a>.</em>

**Comment by trdm** [comment]
Its workw on Windows 7 with https://github.com/crystalidea/qt6windows7
Thank you very much!
